cypress/integration/routes.spec.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+describe('Routes', () => {
+  it('visit all routes', () => {
+    cy.visit('/')
+
+    cy.findByText(/feedback/i).click()
+    cy.findByText(/Raids/i).click()
+    cy.get('h2')
+      .should('contain.text', 'Active')
+      .should('contain.text', 'Completed')
+      .should('exist')
```

Comment:
this seems redundant if you get the element and elements contain those texts

Reply:
when i was writing that test i was thinking A11y.  but now i think i should have not

Reply:
This has nothing to do with A11y it's a logical redundancy "if the thing exists and I find stuff inside, also make sure it exists"  if you found it and it has stuff... 
Then it exists.